### PR TITLE
🌐 Add French translation for `deployment/versions.md`

### PR DESCRIPTION
**Pull request by rjNemo**
🤗 

Here is the PR to translate the deployment versions page.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews

### CODE REVIEW

File: docs/fr/docs/deployment/versions.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## À propos des versions
+
+Suivant les conventions de versionnage sémantique, toute version inférieure à `1.0.0` peut potentiellement ajouter 
+des changements non rétrocompatibles.
+
+FastAPI suit également la convention que tout changement de version "PATCH" est pour des corrections de bogues et 
+des changements rétrocompatibles.
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/versions.md:51]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/versions.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Donc, vous devriez être capable d'épingler une version comme suit :
+
+```txt
+fastapi>=0.45.0,<0.46.0
+```
+
+Les changements non rétrocompatibles et les nouvelles fonctionnalités sont ajoutés dans les versions "MINOR".
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/versions.md:62]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/versions.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    Le "MINOR" est le numéro au milieu, par exemple, dans `0.2.3`, la version MINOR est `2`.
+
+## Mise à jour des versions FastAPI
+
+Vous devriez tester votre application.
+
+Avec **FastAPI** c'est très facile (merci à Starlette), consultez la documentation : [Testing](../tutorial/testing.md){.
+internal-link target=_blank}
```
Comment (Smlep) [inline docs/fr/docs/deployment/versions.md:70]: The linebreak breaks the link, see the built [docs](https://61220d5a3aa06f8654ae3335--fastapi.netlify.app/fr/deployment/versions/#mise-a-jour-des-versions-fastapi), the link does not work

File: docs/fr/docs/deployment/versions.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Vous ne devriez pas épingler la version de `starlette`.
+
+Différentes versions de **FastAPI** utiliseront une version spécifique plus récente de Starlette.
+
+Ainsi, vous pouvez simplement laisser **FastAPI** utiliser la bonne version de Starlette.
+
+## À propos de Pydantic
+
+Pydantic inclut des tests pour **FastAPI** avec ses propres tests, ainsi les nouvelles versions de Pydantic (au-dessus 
+de `1.0.0`) sont toujours compatibles avec FastAPI.
```
Comment (Smlep) [inline docs/fr/docs/deployment/versions.md:88]:
```suggestion
de `1.0.0`) sont toujours compatibles avec **FastAPI**.
```

**Review by Smlep — CHANGES_REQUESTED** [review]

**Review by Smlep — APPROVED** [review]

**Review by axel584 — APPROVED** [review]
J'aurai peut-être utilisé le terme "figer une version" plutôt que "épingler une version"... Mais c'est un avis personnel.

**Comment by tiangolo** [comment]
Great, thanks @rjNemo! :cookie: 

And thanks for the reviews @axel584 and @Smlep! :bow:

**Pull request merged**